### PR TITLE
docs: add docs about the demo application

### PR DESCRIPTION
**Pull request by smoya**
**Description**
It closes https://github.com/asyncapi/event-gateway/issues/23.

This PR adds documentation about the recently deployed Event-Gateway demo application, including a video recording I made today.

**Related issue(s)**
https://github.com/asyncapi/event-gateway/issues/23

### CODE REVIEW

**Review by derberg — DISMISSED** [review]

**Comment by derberg** [comment]
now I know where "the `open in Studio` for markdown" idea came from  😄

**Comment by smoya** [comment]
Pinging @alequetzalli so it would be awesome if you can do a brief review to this!

File: README.md [hunk 1]
```diff
@@ -70,6 +70,51 @@ However, we reduced the scope for the first versions, so we can give support to
...
+    lumens:
+      type: integer
+      minimum: 0
+      description: Light intensity measured in lumens.
+    sentAt:
+      type: string
+      format: date-time
+      description: Date and time when the message was sent.
+  ```
+  You can use [kcat](https://github.com/edenhill/kcat) as Kafka producer (headers are totally optional):
```
Comment (alequetzalli) [inline README.md:98]:
```suggestion
  You can use [kcat](https://github.com/edenhill/kcat) as the Kafka producer (headers are totally optional):
```

File: README.md [hunk 2]
```diff
@@ -70,6 +70,51 @@ However, we reduced the scope for the first versions, so we can give support to
...
+  ```bash
+  echo '{"lumens": 100}' | kcat -H test=true -H origin=readme -H time=(date) -b event-gateway-demo.asyncapi.com:20472 -t event-gateway-demo -P
+  ```                
+- **Consume** Kafka messages from the Kafka broker `event-gateway-demo.asyncapi.com:20472` and topic `event-gateway-demo`.
+  You can use [kcat](https://github.com/edenhill/kcat) as Kafka consumer (headers are totally optional):
```
Comment (alequetzalli) [inline README.md:103]:
```suggestion
  You can use [kcat](https://github.com/edenhill/kcat) as the Kafka consumer (headers are totally optional):
```

**Review by alequetzalli — COMMENTED** [review]
@smoya added my review 🙂

**Review by alequetzalli — APPROVED** [review]
LGTM 🚀

**Review by derberg — APPROVED** [review]
:shipit:

**Pull request merged**